### PR TITLE
feature: retrieve access token

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
Miniapp can get the access token from hostapp.
Able to provide the validation check between hostapp and miniapp based on miniapp id. HostApp can throw exception to reject the request of access token.

Note: As discussion, the user permission for access token will be implemented when first time permission is done.

## Links
MINI-1863

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: miniapp/USERGUIDE.md [hunk 1]
```diff
@@ -163,21 +163,31 @@ val miniAppMessageBridge = object: MiniAppMessageBridge() {
...
-        val name: String = ""
+        var name: String = ""
         // Implementation details to get user name
         // .. .. ..
         return name
     }
 
     override fun getProfilePhoto(): String {
-        val profilePhotoUrl: String = ""
+        var profilePhotoUrl: String = ""
```
Comment (khairul-alam-licon) [inline miniapp/USERGUIDE.md:177]: Ah yes, right :D

**Review by khairul-alam-licon — APPROVED** [review]
I think, it looks good 👍

**Review by corycaywood — APPROVED** [review]
Looking good 👍  Just a few minor comments.

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridgeDispatcher.kt [hunk 2]
```diff
@@ -17,20 +19,16 @@ abstract class UserInfoBridgeDispatcher {
...
-        throw MiniAppSdkException(
-            "The `UserInfoBridgeDispatcher.getProfilePhoto`" +
-                    " method has not been implemented by the Host App."
-        )
-    }
+    open fun getProfilePhoto(): String =
+        throw MiniAppSdkException("The `UserInfoBridgeDispatcher.getProfilePhoto` $NO_IMPL")
+
+    /** Get access token from host app. **/
+    open fun getAccessToken(miniAppId: String): TokenData =
```
[the remark this answers is not in the document]
Comment (ts-leojoseph-r) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridgeDispatcher.kt:30]: Guess, I need to the same in iOS 😉

**Pull request merged**